index.d.ts

```diff
@@ -22,6 +22,13 @@ declare namespace elementReady {
 		@default true
 		*/
 		readonly stopOnDomReady?: boolean;
+
+		/**
+		Explicitly wait for entire element to be loaded instead of just its opening HMTL tag. If this is set to false, a `elementReady('nav')` promise could resolve before the last menu item has been downloaded. Note: This is unrelated to the loading of images, videos, scripts, etc. It only follows the loading of the current document.
+
+		@default true
+		*/
+		readonly expectEntireElement?: boolean;
```

Comment:
Not sure about the word "expect". How about `waitForEntireElement`?

Reply:
Alternatively, to also hint that we're dealing with HTML:

- `justTheOpeningTag: false` (reversed)
- `waitForClosingTag: true`

Reply:
> waitForClosingTag: true

I'm not sure it's clear enough that it means it will wait for the child contents too. Like, it makes sense when you think about it, but it was not immediately clear.

How about any of these:
- `waitForSubtree`
- `waitForChildren`
- `waitForChildElements`

Reply:
`waitForChildren` isn't too bad. Technically it's `ChildNodes` (`element.children` is only `Element`s) but it's understandable enough.

This also matches the new description I suggested below

Reply:
Let's go with `waitForChildren` then.